.npmrc

```diff
@@ -1,5 +1,6 @@
-@bentley:registry=https://registry.npmjs.org/
+@bentley:registry=https://pkgs.dev.azure.com/bentleycs/_packaging/Packages/npm/registry/
```

Comment:
Are we sure about adding this in this repo?

Reply:
No. I thought the classification systems' schema is only available on internal registry, but I see it in npm too, so the change shouldn't be necessary. reverted